examples/async_transcribe_local_media_file.js

```diff
@@ -27,7 +27,8 @@ const token = require('./config/config.json').access_token;
         }],
         filter_profanity: false,
         remove_disfluencies: false,
-        delete_after_seconds: 2592000 // 30 days in seconds
+        delete_after_seconds: 2592000, // 30 days in seconds
+        language: "en" // ISO 639-1 2-letter language code. 
```

Comment:
As we discovered in the manual testing, if someone modifies the example to use a language other than English the API will return a 400 response. I think we should do two things:
1. comment out the extra parameters with explanation:

```
// filter_profanity: false // optional value available with some languages
...
```

2. We can wrap the api execution with a try catch and print the error object to make exception details more clear.